Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewStyles.xaml

```diff
@@ -7,6 +7,10 @@
         <VirtualizingStackPanel Orientation="Horizontal" />
     </ItemsPanelTemplate>
 
+    <ItemsPanelTemplate x:Key="VerticalListItemsPanel">
+        <VirtualizingStackPanel Orientation="Vertical" />
```

Comment:
This should be an `ItemsStackPanel` rather than a `VirtualizingStackPanel`.

Reply:
Changed!.